README.md

```diff
@@ -25,7 +25,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
 
-<img src="figures/dependencies.png" width="677" height="212" />
+<img src="figures/dependencies.png" />
```

Comment:
@rpominov added the dimensions in #119 for a good reason. We should specify the new dimensions to avoid regressing.